pkg/lib/cockpit-components-terminal.tsx

```diff
@@ -347,18 +436,16 @@ export class Terminal extends React.Component {
         this.setState(this.calculateDimensions());
     }
 
-    setTerminalTheme(theme) {
+    setTerminalTheme(theme: TerminalTheme) {
         this.terminal.options.theme = themes[theme];
     }
 
-    onBeforeUnload(event) {
+    onBeforeUnload(event: Event) {
         // Firefox requires this when the page is in an iframe
         event.preventDefault();
 
-        // see "an almost cross-browser solution" at
-        // https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event
-        event.returnValue = '';
-        return '';
+        // Included for legacy support, e.g. Chrome/Edge < 119
+        event.returnValue = true;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21680-b536158f-20250901-092105-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)